### ISSUE TITLE
Refresh rate is not displayed on the scrcpy screen when refresh rate is on in the developement.

### ISSUE THREAD

**Issue by ohsukkwon** [Genymobile/scrcpy#5664]
When the RefreshRate feature is turned on in the Development options, a number is displayed on the top-left corner of the screen, but that number is not visible on the scrcpy screen.

![20241213_132835](https://github.com/user-attachments/assets/03ec4c8d-76ad-4d56-ae1c-78c2fdc97e14)

**Comment by gaberilde** [comment]
i think the number is displayed directly with the graphics/controlled by system and scrpy cant display it

i could be wrong but if not there probably isnt a real fix

**Comment by ohsukkwon** [comment]
Thanks for your reply kindly.
I know that it has displayed on previous OS(like T os, S os...), but it's not displayed on U os.
So I want to find any solution for it.

![LF_V2_20241218_054518_R39N4064TNH_F700NKOS4KWL1](https://github.com/user-attachments/assets/3df2b5f9-e897-48dc-9fcc-551a249abb5b)

**Comment by rom1v** [comment]
Yes, there is probably nothing scrcpy can do to capture this layer.

But is it the value your really want? There are 3 fps values:
 - the number of frames rendered by Android per second
 - the number of frames captured for encoding per second (hopefully, all rendered frames are captured, but this is not necessarily the case if the encoder is too slow for example with a software encoder)
 - the number of frames displayed on the client per second (logged with `--print-fps`)
